beam/www/beam/pages/JobCard.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <h1>Job Cards</h1>
+    <div @click="redirectToJobOperations(operationID = operation)" v-for="operation in operations">
+        <li><span>
+                {{ operation.operationID }}
+            </span>
+            <span class="right-align">
+                {{ operation.completedOperations }}/{{ operation.maxOperations }}
+            </span>
+        </li>
+    </div>
```

Comment:
I think you can use Vue-Router for this instead of defining a function, but test it out:

```suggestion
    <router-link v-for="operation in operations" :to="{ name: 'operation', params: { operationID: operation.operationID } }">
        <li>
        	<span>
                {{ operation.operationID }}
            </span>
            <span class="right-align">
                {{ operation.completedOperations }}/{{ operation.maxOperations }}
            </span>
        </li>
    </router-link>
```